helm/templates/deployment.yaml

```diff
@@ -12,6 +12,8 @@ spec:
     metadata:
       labels:
         app.component: {{ .Values.container.name }}
+      annotations:
+        redeployOnChange: "{{ .Values.container.redeployOnChange }}"
```

Comment:
Does this change relate to the auth feature? Happy for it go through, just not sure how it relates.

Reply:
Helm doesn't redeploy an image unless something in the config has changed, which is a problem for dev when the image name is the same. I find this useful locally, and is requried by the pipeline once it is set up, it's OK to stay in this PR I think.